internal/commands/whoami/command.go

```diff
@@ -51,5 +76,23 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 	}
 
 	ui.Print(terminal.NewTextLog("Currently logged in user: %s (%s)", userDisplay, userSecret))
+
+	if cmd.inputs.ShowProjects {
+		if groups, err := clients.Atlas.Groups(); err == nil {
```

Comment:
I think we want to merge in #299 first and then update this to use the new `atlas.AllGroups` function